src/main/java/net/mcreator/io/OS.java

```diff
@@ -62,4 +62,15 @@ public static String getRuntimeProvider() {
 		}
 	}
 
+	public static String getArchitecture() {
+		String osArch = System.getProperty("os.arch");
+		if (osArch.contains("x86_64") || osArch.contains("amd64")) {
+			return "x86_64";
+		} else if (osArch.contains("aarch64")) {
+			return "aarch64";

```

Comment:
Architecture names should be constants (similar to OS names in this file), so the string is not inlined all over the place.